src/main/java/school/lemon/changerequest/java/introduction/hw2/Task8.java

```diff
@@ -1,8 +1,43 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
+import com.sun.tools.javac.util.ArrayUtils;
```

Comment:
please remove this import, cause due to this import your code could not be compiled.